.github/workflows/e2e.yml

```diff
@@ -0,0 +1,140 @@
+name: E2E Suite
```

Comment:
Does it make sense to call this our "E2E suite"? Are you thinking we would introduce other end to end tests here? Otherwise, I'd prefer that we call these our "Kubernetes Integration Tests", like our other integration tests.

Reply:
I have some plans on including Test Harness here too eventually. This needs a proper discussion of course, but it makes a lot of sense to me. Test Harness and K8s tests can share the release binaries, and, generally, they are of the same nature - in essence - end-to-end tests.

Reply:
That makes more sense.